doc/modules/language-guide/pages/local-objects-classes.adoc

```diff
@@ -353,7 +365,7 @@ The constituents of the body marked `public` contribute to the resulting objects
 
 =====  Another example: `Bits`
 
-As another example, let's consider the task of walking the bits of a `Nat`ural number. For this example, we could define the following:
+As another example, let's consider the task of walking the bits of a Natural number. For this example, we could define the following:
```

Comment:
Before:
<img width="887" alt="Screenshot 2021-12-11 at 3 18 03 PM" src="https://user-images.githubusercontent.com/20852629/145674539-72dffa5c-450d-4ed5-9be2-b51dc1e82a83.png">

After:
![Uploading Screenshot 2021-12-11 at 3.18.31 PM.png…]()

Reply:
```suggestion
As another example, let's consider the task of walking the bits of a natural number (type `Nat`). For this example, we could define the following:
```

Reply:
Host about this?

Reply:
I think that's a great suggestion 👍